projects/pgai/uv.lock

```diff
@@ -2053,7 +2053,6 @@ wheels = [
 
 [[package]]
 name = "pgai"
-version = "0.5.0"
```

Comment:
when i ran `uv sync --all-extras` this was removed, I didn't do that 😅

Reply:
Ya I don't quite get what's going on there. I suspect that different minor versions (0.4 vs 0.5) of uv treat this parameter differently, and some write it in and others remove it.